From 50d1181ed4a41685728f34907104fb31a98301ba Mon Sep 17 00:00:00 2001
From: Valentin Raevsky <valentin@compulab.co.il>
Date: Sun, 12 May 2013 16:48:09 +0300
Subject: ARM: i.MX6: cm-fx6: turn on analog sound support in defconfig

Turn on analog sound sound support in defconfig for the cm-fx6 module.
---
 arch/arm/configs/cm_fx6_defconfig |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm/configs/cm_fx6_defconfig b/arch/arm/configs/cm_fx6_defconfig
index 01188e4..e27c020 100644
--- a/arch/arm/configs/cm_fx6_defconfig
+++ b/arch/arm/configs/cm_fx6_defconfig
@@ -166,6 +166,7 @@ CONFIG_IMX2_WDT=y
 CONFIG_MFD_WM8994=y
 CONFIG_MFD_PFUZE=y
 CONFIG_REGULATOR=y
+CONFIG_REGULATOR_DUMMY=y
 CONFIG_REGULATOR_FIXED_VOLTAGE=y
 CONFIG_REGULATOR_PFUZE100=y
 CONFIG_DRM=m
@@ -190,6 +191,7 @@ CONFIG_SND=y
 CONFIG_SND_USB_AUDIO=y
 CONFIG_SND_SOC=y
 CONFIG_SND_IMX_SOC=y
+CONFIG_SND_SOC_CM_FX6=y
 CONFIG_SND_SOC_IMX_SPDIF=y
 CONFIG_SND_SOC_IMX_HDMI=y
 CONFIG_HIDRAW=y
-- 
1.7.9.5

